.github/workflows/tests.yml

```diff
@@ -60,6 +60,8 @@ jobs:
         ports:
           - 5432:5432
     steps:
+    - name: Apt update
+      run: sudo apt-get update
```

Comment:
(super mega nitpick (sorry!) - it's generally fine to just use `apt update` nowdays, safes a few keystrokes :)

Reply:
All the other statements use apt-get so it was just for uniformity more than anything :innocent: